ignite/metrics/metrics_lambda.py

```diff
@@ -0,0 +1,55 @@
+from ignite.metrics.metric import Metric
+
+
+class MetricsLambda(Metric):
+    """
+    Apply a function to other metrics to obtain a new metric.
+    The result of the new metric is defined to be the result
+    of applying the function to the result of argument metrics.
+
+    When update, this metric does not recursively update the metrics
+    it depend on. When reset, all its dependency metrics would be
+    resetted. When attach, all its dependencies would be automatically
+    attached.
+
+    Arguments:
+        f (callable): the function that defines the computation
+        args (sequence): Sequence of other metrics or something
+            else that will be fed to ``f`` as arguments.
+
+    Examples::
```

Comment:
```suggestion
    Examples:
```

Reply:
@vfdev-5 I think this should be double colon, just like in:
https://pytorch.org/docs/stable/_modules/torch.html#set_default_dtype

Reply:
Maybe both works but in `ModelCheckpoint` we have a similar case with single colon:
https://pytorch.org/ignite/_modules/ignite/handlers/checkpoint.html#ModelCheckpoint

Reply:
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks

> A paragraph consisting of two colons ("::") signifies that the following text block(s) comprise a literal block. The literal block must either be indented or quoted (see below). No markup processing is done within a literal block. It is left as-is, and is typically rendered in a monospaced typeface:

I think both works. Do you want to change to `:`?